Dockerfile

```diff
@@ -1,16 +1,10 @@
-FROM praekeltfoundation/python-base
+FROM praekeltfoundation/supervisor
 MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
 
-RUN apt-get-install.sh gcc
-RUN apt-get-install.sh python-pip
-RUN apt-get-install.sh python-dev
-RUN apt-get-install.sh libjpeg-dev
-RUN apt-get-install.sh zlib1g-dev
-RUN pip install vxyowsup
+RUN apt-get-install.sh libjpeg62
+RUN apt-get-install.sh nginx
```

Comment:
Each time `apt-get-install.sh` runs, it does an `apt-get update` which can be pretty slow. Generally better to do all the installs in one line.